brands/BOM/tokens/index.js

```diff
@@ -493,4 +493,33 @@ export default {
 		},
 		marginBottom: '18px',
 	},
+
+	modal: {
+		backgroundColor: '#fff',
+		borderRadius: 3,
+		width: {
+			small: 300,
+			medium: 600,
+			large: 900,
+		},
```

Comment:
What I've been trying to do with tokens is ensure that the last key used in each token setting is always a CSS property. That way it's more consistent, and can be also spread. I reckon this should be like this, which is more consistent with other component's tokens with size (ie. button)...

```
modal: {
	size: {
		small: {
			width: 300,
		},
		medium: {
			width: 600,
		},
		large: {
			width: 900,
		},
	},
	...
},
```